manifests/database/postgres-db-pv.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolume                        # Create a PV
+metadata:
+  name: hasher-postgres-pv
+  labels:
+    type: local
+    app: kuber-hasher
+spec:
+  storageClassName: manual                    # Storage class. A PV Claim requesting the same storageClass can be bound to this volume.
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteMany
+  hostPath:                                   # hostPath PersistentVolume is used for development and testing. It uses a file/directory on the Node to emulate network-attached storage
+    path: "/mnt/data"
+  persistentVolumeReclaimPolicy: Retain       # Retain the PV even after PVC is deleted. 
```

Comment:
No new line before EOF )

You can check it by yourself on the github )
<img width="139" alt="Screenshot 2022-07-21 at 12 19 43" src="https://user-images.githubusercontent.com/16806883/180178818-c255172b-ce27-4966-aab1-eb67dfa263da.png">

Reply:
Fixed, added new line before EOF